guide/settings/production.py

```diff
@@ -1,7 +1,7 @@
 from .base import *
 
 DEBUG = False
-
+STATICFILES_STORAGE = "django.contrib.staticfiles.storage.ManifestStaticFilesStorage"
```

Comment:
Can we keep this in base? Just override `STATICFILES_STORAGE` in dev or test settings if you need something else. 

At least keep it with the comment.

Reply:
So I was facing some issues while running tests and I asked on support and Matt suggested to comment this out in base and re-add it in production. Here is the link to the [discussion](https://wagtailcms.slack.com/archives/C81FGJR2S/p1655212725331549)